README.md

```diff
@@ -40,47 +40,57 @@ cf set-env conjur-service-broker SECURITY_USER_PASSWORD [value]
 ```
 
 To configure the Service Broker to communicate with your external Conjur instance, the Service Broker app requires the following environment variables:
-- `CONJUR_VERSION`: The version of your Conjur instance (`4` or `5`); defaults to 5.
-- `CONJUR_ACCOUNT`: The account name for the Conjur instance you are connecting to.
-- `CONJUR_APPLIANCE_URL`: The URL of the Conjur appliance instance you are connecting to. If using high availability, this should be the URL for the Master host in the cluster. This is the URL that the service broker will use to communicate with Conjur.
+- `CONJUR_VERSION`: the version of your Conjur instance (`4` or `5`); defaults to 5.
+- `CONJUR_ACCOUNT`: the account name for the Conjur instance you are connecting to.
```

Comment:
I wonder if we should put a note here about giving the URL of the master load balancer if using HA